envs/python-notebook/environment.yml

```diff
@@ -0,0 +1,24 @@
+name: carbonplan
+channels:
+  - conda-forge
+dependencies:
+  - adlfs
+  - dask[complete]
+  - dask-kubernetes
+  - fsspec
+  - google-cloud-storage
+  - gcsfs
+  - gstools
+  - intake
+  - intake-esm
+  - intake-xarray
+  - prefect
+  - regionmask
+  - scikit-downscale
+  - scikit-learn
+  - tenacity
+  - xarray
+  - xesmf
+  - xpersist
+  - xclim
+  - zarr
```

Comment:
Also add the contents of the `requirements-dev.txt` file here. And you can move the scikit-downscale pin from the github action to here: 

```yaml
  - pip
  - pip:
      - git+https://github.com/pangeo-data/scikit-downscale@84cd5f45de4aa61445ca74b85505b035f84fbb80
```